python/graphstorm/trainer/gsgnn_trainer.py

```diff
@@ -192,11 +192,12 @@ def save_model(self, model, epoch, i, save_model_path):
         '''
         barrier()
         if save_model_path is not None:
-            assert isinstance(model.module, (GSgnnModel, GSgnnModelBase)), \
```

Comment:
there are integration tests for the standalone mode. Can you trigger model saving in integration test of standalone mode?

Reply:
here is the integration test. https://github.com/awslabs/graphstorm/blob/main/tests/end2end-tests/graphstorm-nc/test.sh#L30

Reply:
done